extensions/window/internal.m

```diff
@@ -188,7 +188,6 @@ static BOOL window_presstab(AXUIElementRef win, CFIndex tabIndex) {
 
     worked = YES;
 cleanup:
-    if (tab) CFRelease(tab);
```

Comment:
Per the first item in this review `tab` was obtained from an array with a `Get` function, so we don't own its memory.